tests/testthat/test-l_fmt_date_time.R

```diff
@@ -318,3 +318,12 @@ test_that("fmt_datetime() works correctly", {
     )
   )
 })
+
+test_that("A gt table will properly output a date to LaTeX out of the box", {
+  skip()
```

Comment:
I attempted to fix an issue where a date would show badly as LaTeX, but I didn't figure it out. So added a test in the meantime.

`as_latex(tab %>% fmt_date(date)` works.